WeartGloves/README.md

```diff
@@ -0,0 +1,39 @@
+# Instruction on how to install and use the Weart Glove Device
+
+## Build and Install
+
+1. Install the `WeArtGloveLib` following the instruction [here](https://github.com/ami-iit/WEART-SDK-Cpp/tree/devel_ami-iit#ami-iit).
+2. clone the repo `git clone https://github.com/ami-iit/yarp-device-haptic-gloves.git`
+3. `git checkout devel_WeartGlove`
+4. `mkdir && cd build`
+5. `cmake -G"Visual Studio 17 2022" ..`
+6. Notice that the files are going to be installed to `C:\Program Files (x86)\YOUR_PROJECT_NAME\bin`
+    - If you want to change `CMAKE_INSTALL_PREFIX`, run `ccmake ..`
+    - In our case: `C:<PATH_TO_ROBOTOLOGYSUPERBUILD>\build\install`
```

Comment:
A few comments here:
- I would just suggest to specify the ``CMAKE_INSTALL_PREFIX`` 
- If this is supposed to work only on Windows, I would specify it. If not, I would prefer not to specify OS specific paths
- I would suggest mentions to "our" case. Who is us? 😉